src/jobs/build_multiplatform_docker.yml

```diff
@@ -90,27 +86,16 @@ steps:
           --build-arg BUILD_NUMBER=$APP_VERSION \
           --build-arg GIT_REF=$CIRCLE_SHA1 \
           --tag "${IMAGE_NAME}:${APP_VERSION}" \
+          --tag "${IMAGE_NAME}:latest" \
           --label "maintainer=dps-hmpps@digital.justice.gov.uk" \
           --label "app.version=${APP_VERSION}" \
           --label "build.version=${APP_VERSION}" \
           --label "build.number=${CIRCLE_BUILD_NUM}" \
           --label "build.url=${CIRCLE_BUILD_URL}" \
           --label "build.gitref=${CIRCLE_SHA1}" \
+          --push \
```

Comment:
multi platform images aren't loaded into the local docker cache as it doesn't support multiple platforms.
Only solution therefore is to combine the publish and the build steps.